akka-http-core/src/main/scala/akka/http/impl/settings/ConnectionPoolSettingsImpl.scala

```diff
@@ -22,7 +24,8 @@ private[akka] final case class ConnectionPoolSettingsImpl(
   idleTimeout:                       Duration,
   connectionSettings:                ClientConnectionSettings,
   poolImplementation:                PoolImplementation,
-  responseEntitySubscriptionTimeout: Duration)
+  responseEntitySubscriptionTimeout: Duration,
+  hostMap:                           immutable.Map[Regex, ConnectionPoolSettings])
```

Comment:
Why is that a map? It seems the mapping feature isn't used anywhere (but `find` instead on key-value pairs). Could be just a `Seq` of key-value pairs.

Reply:
Honestly I don't know why it's a map. If I made it a seq, would that address your concern about order? Does the typesafe config allow ordered retrieval in this way?